kubernetes/configmaps.yaml

```diff
@@ -0,0 +1,34 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: crdt-config
+  namespace: coordnet
+data:
+  HOCUSPOCUS_PORT: "8010"
+  HOCUSPOCUS_TIMEOUT: "30000"
+  HOCUSPOCUS_DEBOUNCE: "2000"
+  HOCUSPOCUS_MAXDEBOUNCE: "10000"
+  HOCUSPOCUS_QUIET: "false"
+  BACKEND_URL: "http://django:5000"
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: django-config
+  namespace: coordnet
+data:
+  USE_DOCKER: "no"
+  REDIS_URL: "redis://redis:6379/0"
+  DJANGO_SETTINGS_MODULE: "config.settings.production"
+  DJANGO_ALLOWED_HOSTS: "localhost,django"
+  DJANGO_SECURE_SSL_REDIRECT: "False"
+  SENTRY_ENVIRONMENT: "production"
+  CONN_MAX_AGE: "0"
+  MAILGUN_DOMAIN: "mail.coord.dev"
```

Comment:
Users will have trouble with mail delivery if they don't change this, should we use a more obvious placeholder? Seems like a silently failing issue.